### DISCUSSION TITLE
Error, while trying to add a new platform

### DISCUSSION THREAD

**Discussion by lokki11199** [The-OpenROAD-Project/OpenROAD#3368] [Q&A]
Hello,
I am trying to integrate a FreePDK45 platform and get an error on global route stage:
![изображение](https://github.com/The-OpenROAD-Project/OpenROAD/assets/131533616/9fe83ce2-8516-4796-b9da-9471ec1e6ca2)

A FreePDK45 platform and Nangate45 are similar to each other, excepting tha geometry of standard cells (in FreePDK45 they are polygon). I am making config.mk based on Nangate45 config.mk.
OpenROAD Flow version: v2.0-7988-g20b907452 

Help me please to solve this problem.

[global_route_PWM_freepdk45nm_base_2023-05-23_00-37.tar.gz](https://github.com/The-OpenROAD-Project/OpenROAD/files/11537324/global_route_PWM_freepdk45nm_base_2023-05-23_00-37.tar.gz)

**Comment by maliberty** [comment]
A crash should be filed as an issue rather than a test case.  I've opened https://github.com/The-OpenROAD-Project/OpenROAD/issues/3369

**Reply by lokki11199** [reply]
Thanks!
